rotkehlchen/chain/evm/decoding/compound/v3/decoder.py

```diff
@@ -209,10 +251,86 @@ def _decode_repay_or_borrow_event(self, context: DecoderContext) -> DecodingOutp
                 break
         else:
             log.error(
-                f'Could not find any compound v3 repay or borrow event in tx '
+                f'Could not find any compound v3 withdraw or borrow event in tx '
                 f'{context.transaction.tx_hash.hex()}.',
             )
 
+        action_items = []  # also create an action item for the spend of the cTokens
+        if paired_event is not None and action_from_event_type is not None:
+            action_items.append(ActionItem(
+                action='transform',
+                from_event_type=action_from_event_type,
+                from_event_subtype=HistoryEventSubType.NONE,
+                asset=compound_token,
+                to_event_subtype=action_to_event_subtype,
+                to_notes=action_to_notes,
+                to_counterparty=CPT_COMPOUND_V3,
+                paired_event_data=(paired_event, False),
+            ))
+
+        return DecodingOutput(action_items=action_items)
+
+    def _decode_collateral_movemement(
+            self,
+            context: DecoderContext,
+            compound_token: EvmToken,
+    ) -> DecodingOutput:
+        """Decode a compound v3 supply/withdraw collateral events"""
+        collateral_asset = EvmToken(evm_address_to_identifier(
+            address=hex_or_bytes_to_address(context.tx_log.topics[3]),
+            chain_id=self.evm_inquirer.chain_id,
+            token_type=EvmTokenKind.ERC20,
+        ))
+        collateral_amount = asset_normalized_value(
+            amount=hex_or_bytes_to_int(context.tx_log.data),
+            asset=collateral_asset,
+        )
+
+        for event in context.decoded_events:
+            if (
+                event.event_type in {HistoryEventType.SPEND, HistoryEventType.RECEIVE} and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.balance.amount == collateral_amount and
+                event.asset == collateral_asset and
+                event.address == compound_token.evm_address
+            ):
+                transfer_event = event
+                event.counterparty = CPT_COMPOUND_V3
+                if event.event_type == HistoryEventType.SPEND:
+                    notes_action = 'Enable'
+                    event.event_type = HistoryEventType.DEPOSIT
+                    event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                    event.notes = f'Deposit {collateral_amount} {collateral_asset.symbol} into compound v3'  # noqa: E501
```

Comment:
Can you change here and in the other places `compound v3` to `Compound v3`. TO match what we do in aave